lib/plugins/aws/provider/awsProvider.test.js

```diff
@@ -398,20 +436,56 @@ describe('AwsProvider', () => {
           },
         }));
 
-      return awsProvider.getServerlessDeploymentBucketName(options.stage, options.region)
+      return awsProvider.getServerlessDeploymentBucketName()
         .then((bucketName) => {
           expect(describeStackResourcesStub.called).to.be.equal(false);
           expect(bucketName).to.equal('custom-bucket');
           awsProvider.request.restore();
         });
     });
-  });
 
-  describe('#getStackName', () => {
-    it('should return the stack name', () => {
-      serverless.service.service = 'myservice';
+    describe('#getStage', () => {
```

Comment:
Is this a correct name? Seems like there's a pair of `()` missing or the `#` needs to be removed.